### DISCUSSION TITLE
Android input error

### DISCUSSION THREAD

**Discussion by Zhou-Jiangcheng** [rustdesk/rustdesk#1671] [General]
When using rustdesk in android to control a PC with Linux installed (Ubuntu 22.04), the input is always problematic.
Each time you press a key, you press the key twice. Just like this picture
![image](https://user-images.githubusercontent.com/39516706/194070827-790d6720-5fac-4d94-b68f-b4734773336c.png)
I use a keyboard with Bluetooth in my android pad.

**Comment by Zhou-Jiangcheng** [comment]
An the mouse wheel can not be used.
I also use a mouse with Bluetooth in my android pad.

**Comment by rustdesk** [comment]
Known and fixing
